#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("color_importance", double_t, 0, "color importance", 0.2, 0.0, 5.0)
gen.add("spatial_importance", double_t, 0, "spatial importance", 0.4, 0.0, 5.0)
gen.add("normal_importance", double_t, 0, "normal importance", 1.0, 0.0, 5.0)
gen.add("use_transform", bool_t, 0, "use transform", True)
gen.add("seed_resolution", double_t, 0, "seed resolution", 0.1, 0.0, 10.0)
gen.add("voxel_resolution", double_t, 0, "voxel resolution", 0.008, 0.0, 1.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "SupervoxelSegmentation"))
